--- a/mix.exs	2021-12-04 15:37:46.879606781 +1000
+++ b/mix.exs	2021-12-04 15:38:24.422368561 +1000
@@ -36,6 +36,7 @@
       releases: [
         pleroma: [
           include_executables_for: [:unix],
+          include_erts: false,
           applications: [ex_syslogger: :load, syslog: :load, eldap: :transient],
           steps: [:assemble, &put_otp_version/1, &copy_files/1, &copy_nginx_config/1],
           config_providers: [{Pleroma.Config.ReleaseRuntimeProvider, []}]
